I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/weekday_run.rpy", line 12, in script
    $ YoIl = day_name[day]
  File "game/weekday_run.rpy", line 12, in <module>
    $ YoIl = day_name[day]
IndexError: list index out of range

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/weekday_run.rpy", line 12, in script
    $ YoIl = day_name[day]
  File "/Applications/renpy-7.1.1-sdk/renpy/ast.py", line 882, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-7.1.1-sdk/renpy/python.py", line 1913, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/weekday_run.rpy", line 12, in <module>
    $ YoIl = day_name[day]
IndexError: list index out of range

Darwin-18.2.0-x86_64-i386-64bit
Ren'Py 7.1.1.929
Nuclear Inssamaker 1.0
Sun Nov 11 14:54:56 2018
